casper/src/test/scala/coop/rchain/casper/api/CreateBlockAPITest.scala

```diff
@@ -50,20 +54,26 @@ class CreateBlockAPITest extends FlatSpec with Matchers {
       "for(_ <- @1){ @2!(2) }"
     ).map(ProtoUtil.sourceDeploy(_, System.currentTimeMillis(), accounting.MAX_VALUE))
 
-    implicit val logEff = new LogStub[Effect]
-    def testProgram(blockApiLock: Semaphore[Effect])(
+    def createBlock(deploy: DeployData, blockApiLock: Semaphore[Effect])(
         implicit casperRef: MultiParentCasperRef[Effect]
-    ): Effect[(DeployServiceResponse, DeployServiceResponse)] = EitherT.liftF(
+    ): Effect[ApiErr[DeployServiceResponse]] =
```

Comment:
Mhh, another type called `Effect`. In my opinion another reason to get rid of the type alias in `BlockAPI.scala` :)